test/e2e/app-dir/metadata-dynamic-routes/index.test.ts

```diff
@@ -280,8 +280,8 @@ createNextDescribe(
       let twitterImageUrlPattern
       if (isNextDeploy) {
         // absolute urls
-        ogImageUrlPattern = /https:\/\/\w+.vercel.app\/opengraph-image\?/
-        twitterImageUrlPattern = /https:\/\/\w+.vercel.app\/twitter-image\?/
+        ogImageUrlPattern = /https:\/\/[\w-]+.vercel.app\/opengraph-image\?/
```

Comment:
Do we want to remove the skipDeployment flag now?

Reply:
removed, but skipped few tests, as they're failing on deployment for now